src/common/apollo/queries.ts

```diff
@@ -37,6 +37,7 @@ export const PRICE_REQUEST_VOTING_DATA = gql`
       orderBy: $orderBy
       orderDirection: $orderDirection
     ) {
+      ancillaryData
```

Comment:
Currently we don't fetch the ancillary data in this request to the graph. I've added the field here because we need the ancillary data to get the title and description for Polymarket requests.